api/routes/orderRoutes.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+
+const { asyncErrorHandler } = require("../middlewares/errorHandler");
+const { createOrder, deleteOrder } = require("../controllers/orderController");
+const { jwtVerify } = require("../middlewares/auth");
+
+const routes = express.Router();
+ 
+routes.post("/items", jwtVerify, asyncErrorHandler(createOrder));
+routes.delete("/", jwtVerify, asyncErrorHandler(deleteOrder));
```

Comment:
```suggestion
routes.delete("", jwtVerify, asyncErrorHandler(deleteOrder));
```
보다 RESTful한 api가 설계될 수 있도록 엔드포인트 끝이 (/)로 작성되는건 지양해주세요 :)

Reply:
넵넵 반영했습니다